edge-info/edge-info

```diff
@@ -46,14 +46,13 @@ _debug(){
     fi
 }
 
-#This bash program is intended to work on multiple build systems.  In yocto/linux micro platform (LMP) we set EDGE_DATA at build time to a user variable (default: /var/rootdirs/userdata) using sed.  For other build systems, follow that technique OR add on to the following function to detect your system and set RFILE and IFILE to your desired locations.
-STORAGE=EDGE_DATA
+#This bash program is intended to work on multiple build systems.  In yocto/linux micro platform (LMP) we set /userdata at build time to a user variable (default: /var/rootdirs/userdata) using sed.  For other build systems, follow that technique OR add on to the following function to detect your system and set IFILE to your desired locations.
+
+STORAGE=/userdata
```

Comment:
I think we must leave this as EDGE_DATA, the sed commands probably want it that way?

Reply:
https://github.com/PelionIoT/meta-edge/blob/main/classes/edge.bbclass#LL80C8-L80C8

Reply:
Good catch.